Assignment_0/210601_MdRahbar.md

```diff
@@ -0,0 +1,104 @@
+# Assignment 0
+
+## I am MD RAHBAR, 210601
```

Comment:
xD Yeah, writeups do include details of the writer him/herself. I'll show or you'll read some more formal writeups, and their details are in much cooler fashion.